os/net/ipv6/sicslowpan.c

```diff
@@ -1179,16 +1176,20 @@ uncompress_hdr_iphc(uint8_t *buf, uint16_t buf_size, uint16_t ip_len)
       PACKETBUF_IPHC_BUF[2] >> 4 : 0;
 
     /* Source address - check context != NULL only if SAM bits are != 0*/
+    struct sicslowpan_addr_context *source_context;
```

Comment:
Doesn't this shadow the previous declaration of source_context that this PR added?

Reply:
Forget that comment, it's just Github stating everything is in the same function.